dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -805,7 +805,8 @@
       "properties": {
         "sentenceCount": {
           "type": "integer",
-          "format": "int32"
+          "format": "int32",
+          "description": "The model will try the best to predict this number of sentences in the output summaries."
```

Comment:
nitpick: How about?
```suggestion
          "description": "It controls the maximum number of sentences in the output summaries."
```

Reply:
Thanks for the suggested wording. Actually it is the approximate number, not the maximum number. Updated.